example/app/config.yml

```diff
@@ -61,7 +61,7 @@ default:
   # test configuration ends
 
   cmaDir: '/opt/'
-  sandbox_cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:4
+  sandbox_cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:5
```

Comment:
Does this or anywhere else need an update to the newest release version at all?

Reply:
Yes, I'm going to need to update this reference to a newly published version.    Everywhere else, kes,etc pull the latest CMA release, so that's good to go.

Reply:
Also update all of the client lib references to 1.0.8

Reply:
Updates:  a427ab5d7

Will merge assuming CI passes